ee/desktop/runner/runner_linux.go

```diff
@@ -104,31 +112,98 @@ func (r *DesktopUsersProcessesRunner) userEnvVars(uid string) map[string]string
 		}
 
 		sessionType := strings.Trim(string(typeOutput), "\n")
-		switch sessionType {
-		case "x11":
-			// We need to set DISPLAY
-			xDisplayOutput, err := exec.CommandContext(ctx, "loginctl", "show-session", session, "--value", "--property=Display").Output()
-			if err != nil {
-				level.Debug(r.logger).Log(
-					"msg", "could not get Display from user session",
-					"uid", uid,
-					"err", err,
-				)
-				continue
-			}
+		if sessionType == "x11" {
```

Comment:
I tend to prefer `switch`, but it's  bike shedding, and not worth the change.

Reply:
I preferred it as well, but it was what necessitated the label (so that we could break out of the outer for loop once we found a good session)

Reply:
Seems weird. Feels like a hint at more refactoring. But 🤷, not clearly worth while.